bin/MessagesBundle_de.properties

```diff
@@ -115,6 +115,22 @@ pcbline_width = Leiterbahnbreite:
 PCBpad = PCB Pad
 pcbpad_height = Höhe Pad:
 pcbpad_intw = Durchmesser Pad Bohrung:
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on it.

Reply:
ok